### PR TITLE
additional explanation for SetDeviceOperatingMode

### PR DESCRIPTION
**Pull request by NerijusKondrotas**
additional explanation for SetDeviceOperatingMode

# Checklist

The following aspects have been respected by the author of this pull request, confirmed by both pull request assignee **and** reviewer:

* Changelog update (necessity checked and entry added or not added respectively)
  * [x] @NerijusKondrotas 
  * [x] @maximilianpilz 
  * [x] @antipovi 
  * [x] @jannast 
* README update (necessity checked and entry added or not added respectively)
  * [x] @NerijusKondrotas 
  * [x] @maximilianpilz 
  * [x] @antipovi 
  * [x] @jannast

### CODE REVIEW

**Review by jannast — DISMISSED** [review]

File: src/t2iapi/device/service.proto [hunk 1]
```diff
@@ -27,6 +27,8 @@ service DeviceService {
     /*
     Switch the device into a specific operating mode.
     This manipulation shall result in a Mds having a specific MdsOperatingMode.
+    The manipulated state shall be persistent until a next manipulation call or until the end of the test case
+    execution.
      */
```
Comment (antipovi) [inline src/t2iapi/device/service.proto:32]: what do you think about adding:  "If the device is not able to maintain the static state, it shall return not supported."

Comment (jannast) [reply src/t2iapi/device/service.proto:32]: yes, I would add this sentence too

File: src/t2iapi/device/service.proto [hunk 2]
```diff
@@ -27,6 +27,8 @@ service DeviceService {
     /*
     Switch the device into a specific operating mode.
     This manipulation shall result in a Mds having a specific MdsOperatingMode.
+    The manipulated state shall be persistent until a next manipulation call or until the end of the test case
```
Comment (maximilianpilz) [inline src/t2iapi/device/service.proto:30]: How is a t2iapi server supposed to know the scope of "the test case execution" ?

I do agree with all parts other than
> or until the end of the test case execution

though.

**Review by maximilianpilz — CHANGES_REQUESTED** [review]

File: src/t2iapi/device/service.proto (hunk 2 above)
Comment (NerijusKondrotas) [reply src/t2iapi/device/service.proto:30]: team decided to remove this part. Done

File: src/t2iapi/device/service.proto (hunk 1 above)
Comment (NerijusKondrotas) [reply src/t2iapi/device/service.proto:32]: added

File: src/t2iapi/device/service.proto (hunk 2 above)
Comment (jannast) [reply src/t2iapi/device/service.proto:30]: as I see, this is not done

**Review by jannast — CHANGES_REQUESTED** [review]
do the decided changes

File: src/t2iapi/device/service.proto (hunk 2 above)
Comment (NerijusKondrotas) [reply src/t2iapi/device/service.proto]: @jannast  [NerijusKondrotas]  committed 6 hours ago.

**Review by maximilianpilz — APPROVED** [review]

**Review by maximilianpilz — CHANGES_REQUESTED** [review]
add a changelog entry that mentions that you changed the semantics

**Review by jannast — APPROVED** [review]
done

**Review by maximilianpilz — DISMISSED** [review]

**Review by antipovi — APPROVED** [review]

**Review by jannast — APPROVED** [review]

**Review by maximilianpilz — APPROVED** [review]

**Pull request merged**